specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json

```diff
@@ -16,6 +16,42 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices": {
+      "get": {
+        "tags": [
+          "BlobService"
+        ],
+        "operationId": "BlobServices_List",
```

Comment:
Please add `"x-ms-pageable": { "nextLinkName": null }` extension to the list operation. It is also done for many other operations like [StorageAccounts_List](https://github.com/Azure/azure-rest-api-specs/blob/c63f7d5f37c4c3be5c065544574b5b32c1ad52aa/specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/storage.json#L323)